buildserver-monitor/http_config.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * \file http_config.h
+ *
+ * \licence "THE BEER-WARE LICENSE" (Revision 42):
+ *          <jilvin.wiegmus@fourtress.nl> wrote this file. As long as you retain
+ *          this notice you can do whatever you want with this stuff. If we
+ *          meet some day, and you think this stuff is worth it, you can buy me
+ *          a beer in return.
+ *                                                                melip muskman
+ *
+ * \brief   Http configuration file for the Buildserver Monitor.
+ *
+ * \details Intended use is to hold the environment specific parts of the Http
+ *          configuration, things that need to be altered for a given buildserver
+ *          environment.
+ *
+ * \date    01-2020
+ */
+
+#ifndef BUILDSERVER_MONITOR_HTTP_CONFIG_H_
+#define BUILDSERVER_MONITOR_HTTP_CONFIG_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/************************************************************************/
+/* Includes                                                             */
+/************************************************************************/
+#include <stdint.h>
+#include <stddef.h>
+
+// Network
+static std::string    USERNAME = "your username";     // The jenkins username.

```

Comment:
For robustness, a check in the httpClient that the default values presented here are not used (but properly filled) would be handy.